ci-operator/config/openshift/kuryr-kubernetes/openshift-kuryr-kubernetes-master.yaml

```diff
@@ -0,0 +1,52 @@
+tag_specification:
+  name: '4.1'
```

Comment:
Sorry about that, but the cut for 4.2 was done yesterday, so this should point to `4.2` now.

Reply:
Not a problem, at least it solves my confusion with 4.1 being there when master is 4.2. ;)